Documentation-AM-3.20 (#948)
gravitee-io/gravitee-docs: pages/am/3.x/user-guide/user-management/password/user-guide-password-policy.adoc

You can set the following password characteristics:

> - *Minimum & Maximum length*: length of the passwords should be enforced by the application.

Reviewer: ```suggestion
- *Minimum & Maximum length*: length of the passwords that should be enforced by the application.
```

- *Numbers*: must include at least one number.
- *Expiry Duration*: the expiration duration (in days) of a password.
- *Special characters*: must include at least one special character.

> - *Mixed case*: must include lower and upper case letters.

Reviewer: ```suggestion
- *Mixed case*: must include lowercase and uppercase letters.
```

- *Exclude common passwords*: will exclude common passwords from a dictionary.

> - *Exclude user profile information from passwords*: will user profile information within passwords (case insensitive).

Reviewer: ```suggestion
- *Exclude user profile information from passwords*: will exclude user profile information from use in passwords (case insensitive).
```

> - *History*: prevent the usage of old password.

Reviewer: ```suggestion
- *History*: prevent the usage of old passwords.
```
